definition/data/sheets/ComplexTypes.json

```diff
@@ -220,6 +220,13 @@
   {"ID": "Applicant", "ListElementCode": "representativeOrganisationDxAddress", "FieldType": "Text", "ElementLabel": "Organisation DxAddress", "SecurityClassification": "Public"},
   {"ID": "Applicant", "ListElementCode": "representativeOrganisationEmail", "FieldType": "Email", "ElementLabel": "Organisation Email", "SecurityClassification": "Public"},
   {"ID": "Applicant", "ListElementCode": "leadApplicantIndicator", "FieldType": "YesOrNo", "ElementLabel": "Lead applicant", "SecurityClassification": "Public"},
+  {"ID": "OtherDirection", "ListElementCode": "extraOrderDirection", "FieldType": "FixedList", "FieldTypeParameter": "ExtraOrderDirectionType", "ElementLabel": "Select direction", "SecurityClassification": "Public"},
+  {"ID": "OtherDirection", "ListElementCode": "extraDocUploadList", "FieldType": "Collection", "FieldTypeParameter": "Text", "ElementLabel": "Extra statements for document upload", "SecurityClassification": "Public", "FieldShowCondition": "extraOrderDirection=\"DOCUMENTS\""},
+  {"ID": "OtherDirection", "ListElementCode": "expertReports", "FieldType": "Collection", "FieldTypeParameter": "Text", "ElementLabel": "Extra statements for expert report", "SecurityClassification": "Public", "FieldShowCondition": "extraOrderDirection=\"EXPERT_REPORT_PERMISSION\""},
+  {"ID": "OtherDirection", "ListElementCode": "directionComment", "FieldType": "TextArea", "ElementLabel": "Extra direction", "SecurityClassification": "Public", "FieldShowCondition": "extraOrderDirection=\"OTHER\""},
```

Comment:
directionComment should be shown only if one of the headers are selected, so the show condition should be '*' (as discussed)

Reply:
👍 got lost somehow in all fo the merges- cheers